packages/config/src/project/utils.ts

```diff
@@ -0,0 +1,342 @@
+import Bluebird from 'bluebird'
+import Debug from 'debug'
+import fs from 'fs-extra'
+import _ from 'lodash'
+import path from 'path'
+
+import { getProcessEnvVars, CYPRESS_SPECIAL_ENV_VARS } from '@packages/server/lib/util/config'
```

Comment:
Any way we can drop pulling in `@packages/server` or `@packages/data-context`?

Reply:
I'm trying. I guess we can remove `@package/server` from here.

Reply:
So excited to see these changes!